admin-ui/app/routes/License/LicenseDetailsPage.js

```diff
@@ -32,146 +24,140 @@ function LicenseDetailsPage({ item, loading, dispatch }) {
-                      value={item.licenseActive ? 'Yes' : 'No'}
-                      isBadge={true}
-                      lsize={3}
-                      rsize={9}
-                      doc_entry="isLicenseActive"
-                      doc_category={LICENSE}
-                    />
-                  </Col>
-                </Row>
-              </Container>
-            ) : (
-              <Alert severity="warning">
-                {!loading &&
-                  'The License API is not enabled for this application.'}
-              </Alert>
-            )}
-          </GluuLoader>
+          {item.licenseEnabled ? (
+            <Container style={{ backgroundColor: '#F5F5F5', float: 'left' }}>
+              <Row>
+                <Col sm={6}>
+                  <GluuFormDetailRow
+                    label="fields.productName"
+                    value={item.productName}
+                    isBadge={true}
+                    lsize={3}
+                    rsize={9}
+                    doc_entry="productName"
+                    doc_category={LICENSE}
+                  />
+                </Col>
+                <Col sm={6}>
+                  <GluuFormDetailRow
+                    label="fields.productCode"
+                    value={item.productCode}
+                    isBadge={true}
+                    lsize={3}
+                    rsize={9}
+                    doc_entry="productCode"
+                    doc_category={LICENSE}
+                  />
+                </Col>
+              </Row>
+              <Row>
+                <Col sm={6}>
+                  <GluuFormDetailRow
+                    label="fields.licenseType"
+                    value={item.licenseType}
+                    isBadge={true}
+                    lsize={3}
+                    rsize={9}
+                    doc_entry="licenseType"
+                    doc_category={LICENSE}
+                  />
+                </Col>
+                <Col sm={6}>
+                  <GluuFormDetailRow
+                    label="fields.licenseKey"
+                    value={item.licenseKey}
+                    isBadge={true}
+                    lsize={3}
+                    rsize={9}
+                    doc_entry="licenseKey"
+                    doc_category={LICENSE}
+                  />
+                </Col>
+              </Row>
+              <Row>
+                <Col sm={6}>
+                  <GluuFormDetailRow
+                    label="fields.customerEmail"
+                    value={item.customerEmail}
+                    isBadge={true}
+                    lsize={3}
+                    rsize={9}
+                    doc_entry="customerEmail"
+                    doc_category={LICENSE}
+                  />
+                </Col>
+                <Col sm={6}>
+                  <GluuFormDetailRow
+                    label="fields.customerName"
+                    value={item.customerFirstName + ' ' + item.customerLastName}
```

Comment:
Prefer to use template literals ```value={`${item.customerFirstName} ${item.customerLastName}`}```

Reply:
I didn't change those lines. Since it contributes to improving the code quality, I will apply the feedback. Thanks!